repos/spack_repo/builtin/packages/cp2k/package.py

```diff
@@ -118,6 +118,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
         " are enabled",
     )
     variant("pytorch", default=False, description="Enable libtorch support")
+    variant("openpmd-api", default=False, description="Enable openPMD support")
```

Comment:
```suggestion
    variant("openpmd-api", default=False, description="Enable openPMD support", when=@2026.1: build_system=cmake)
```
Since this relates to an open PR (https://github.com/cp2k/cp2k/pull/4058), and you only added it to the `CMakeBuilder`, this variant should only be available from the next release `2026.1` onward (note: `master` is considered more recent than any other version, so `cp2k@master +openpmd-api` will be valid with `when=2026.1:`) and only for building with CMake.

Reply:
Since you PR on the CP2K side is still unmerged, to test you will need something like `cp2k@git.bd41001017e7858fecfaf3c3b43d24d1732228eb=master +apenpmd-api`.